tensorflow_recommenders/layers/embedding/tpu_embedding_layer.py

```diff
@@ -330,18 +330,18 @@ class instance you specify will be used to manage the slot variables. This
   strategy = tf.distribute.TPUStrategy(...)
   with strategy.scope():
     embedding_inputs = {
-        'feature_one': tf.keras.Input(batch_size=1024, shape=(),
+        'feature_one': tf.keras.Input(batch_size=1024, shape=(1,),
                                       dtype=tf.int32),
-        'feature_two': tf.keras.Input(batch_size=1024, shape=(),
+        'feature_two': tf.keras.Input(batch_size=1024, shape=(1,),
                                       dtype=tf.int32, ragged=True),
-        'feature_three': tf.keras.Input(batch_size=1024, shape=(),
+        'feature_three': tf.keras.Input(batch_size=1024, shape=(1,),
                                         dtype=tf.int32)}
     # embedding, feature_config and embedding_inputs all have the same nested
     # structure.
     embedding = tpu_embedding_layer.TPUEmbedding(
         feature_config=feature_config,
         optimizer=tf.tpu.experimental.embedding.SGD(0.1))(embedding_inputs)
-    logits = tf.keras.layers.Dense(1)(tf.concat(tf.nest.flatten(embedding)))
+    logits = tf.keras.layers.Dense(1)(tf.concat(tf.nest.flatten(embedding), axis=1))
```

Comment:
Could you please make sure that this line doesn't go over 80 characters?

Reply:
yes will do.